### ISSUE TITLE
Accessibility: close/cancel when escape is pressed

### ISSUE THREAD

**Issue by mollylentz** [tristen/hoverintent#27]
If anyone has figured out a way to make this work, that would be awesome. For accessibility reasons, you should be able to hit the escape key in order to hide the bit that is being revealed on hover. I have yet to figure out an elegant way to do this.

**Comment by bronzehedwick** [comment]
@mollylentz Can you use the `handleFocus` setting to add a callback for onOver/onOut, then add/remove the escape key listener there, respectively, and finally hide then menu?

**Comment by mollylentz** [comment]
That completely worked! I must have really been hitting a wall yesterday. Thank you so very much.
